topics/ecology/tutorials/regionalGAM/Rdata_tutorial.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: tutorial_hands_on
+topic_name: ecology
+tutorial_name: Regional GAM
+---
+
+# Introduction
+{:.no_toc}
+
+⚠️ You might be willing to follow this tutorial if you want to learn how to deal with a dataset which is on RData format.
```

Comment:
Can you avoid the emoji? You can use the icons that are defined in the `_config.yaml` file at the root